src/category_theory/monoidal/category.lean

```diff
@@ -600,6 +600,91 @@ end
 
 end
 
+section
+
+universes v₁ v₂ u₁ u₂
+
+variables (C₁ : Type u₁) [category.{v₁} C₁] [monoidal_category.{v₁} C₁]
+variables (C₂ : Type u₂) [category.{v₂} C₂] [monoidal_category.{v₂} C₂]
+
+instance prod_monoidal : monoidal_category (C₁ × C₂) :=
+{ tensor_obj := λ X Y, (X.1 ⊗ Y.1, X.2 ⊗ Y.2),
+  tensor_hom := λ X₁ Y₁ X₂ Y₂ f₁ f₂, (f₁.1 ⊗ f₂.1, f₁.2 ⊗ f₂.2),
+  tensor_id' :=
+    λ X₁ X₂,
+      congr_arg2 prod.mk
+        (tensor_id X₁.1 X₂.1)
+        (tensor_id X₁.2 X₂.2),
```

Comment:
Also, too many unnecessary newlines here. This should probably be
```suggestion
  tensor_id' := λ X₁ X₂,
    congr_arg2 prod.mk (tensor_id X₁.1 X₂.1) (tensor_id X₁.2 X₂.2),
```
Or indeed all on one line if it fits.